docs/eigenlayer/restaking-guides/1-restaking-developer-guide.md

```diff
@@ -1,8 +1,166 @@
+
+
+
+
+
+# Smart Contract Liquid Restaking User Guide
+
+## Steps to Deposit (Restake) Liquid Tokens
+
+1. Call StrategyManager.depositIntoStrategy() .
+2. User is now actively Restaked.
+
+## Steps to Withdraw (Unstake) Liquid Tokens
+
+1. Queue Withdrawal: call DelegationManager.queueWithdrawal() to trigger the escrow period. Wait for Escrow Period: 24 days for $EIGEN, 7 days for all other tokens. Please see further detail [here](https://docs.eigenlayer.xyz/eigenlayer/restaking-guides/restaking-user-guide/#escrow-period-withdrawal-delay).
+
+2. Complete Withdrawal as Tokens: call DelegationManager.completeQueuedWithdrawal() to complete the withdrawal and return assets to the withdrawer's wallet.
+
+
+# Smart Contract Delegation User Guide
+
+The process of Delegating assets is the same for both liquid and native restaked assets. The user's Restaking wallet must Delegate all restaked assets to a single Operator. After the initial Delegate operation - any subsequent Deposited (Restaked) assets are also automatically delegated to the current operator.
+
+## Steps to Delegate Assets
+
+1. Call DelegationManager.delegateTo()
+2. Your Restaked assets are now delegated.
+
+## Steps to Change Actively Delegated Operator
+
+_Coming Soon_ In the meantime, please see the smart contract specifications above and the high level process walkthrough [here](./0-restaking-user-guide/restaker-delegation/redelegation-process.md).
+
+
+# Smart Contract Native Restaking User Guide
+
+For users planning to restake multiple validators, we recommend they connect many validators to a single EigenPod in order to reduce cost and complexity where possible. "Generate Proof Via eigenpod-proofs-generation CLI" will prove all connected validators.
+
+## PEPE Release
+
+EigenLayer core contracts have had two previous major releases: M1 and M2. PEPE (Protocol: EigenPod Enhancement upgrade) is the current major release that focuses primarily on enhancements to Native Restaking and EigenPod design. The PEPE release will result in significantly lower gas fees and great compatibility with future scheduled Ethereum network upgrades.
+
+Please see the [this document](https://hackmd.io/@-HV50kYcRqOjl_7du8m1AA/SkJPfqBeC) and [PR #515](https://github.com/Layr-Labs/eigenlayer-contracts/pull/515) for a design history and motivation for the PEPE Release.
+
+
+## EigenPod Upgrades
+
+For all M1 to PEPE migrations - we no longer require users to upgrade their EigenPod contracts per the deprecated activateRestaking() method. M1 pods will be upgraded automatically to PEPE compliant EigenPods by EigenLabs.
+
+The delayed withdrawal router will remain functional after PEPE. It will not receive new consensus rewards from EigenPods, however if you have existing rewards you may continue to claim them as they become claimable.
```

Comment:
"The delayed withdrawal router is being deprecated with the PEPE release, but will remain functional."

Reply:
done